2003-12-15  Dmitry V. Levin <ldv@altlinux.org>

	* strace.c (main): Set line buffering for piped output.

--- strace-4.5.1.orig/strace.c	2003-11-12 00:24:23 +0300
+++ strace-4.5.1/strace.c	2003-12-15 00:46:04 +0300
@@ -357,6 +357,7 @@ char *argv[];
 					strerror(errno));
 				exit(1);
 			}
+			setlinebuf(outf);
 		}
 		else if ((outf = fopen(outfname, "w")) == NULL) {
 			fprintf(stderr, "%s: can't fopen '%s': %s\n",
